### PR TITLE
Website home page: add to user quotes #7854

### LINKED ISSUE #7854: Website home page: add to user quotes
Where to add:
image

What to add:
image
`- Faculty user, UK`

image
`- Faculty user, Singapore`

### PR DESCRIPTION
**Pull request by kyleaolson**
Fixes #7854 

Added new user quotes to TESTIMONIALS array and updated initial value on homepage to last value.

### CODE REVIEW

**Comment by damithc** [comment]
@kyleaolson The PR title should match the issue title exactly

**Comment by kyleaolson** [comment]
Looks like the build failed. I'm looking into that and will fix as needed shortly.

**Comment by HirdayGupta** [comment]
@kyleaolson Your PR title is incorrect. It needs to match the title of the Issue you are fixing (you updated it to match the title of the wrong issue). Your PR title needs to be `Website home page: add to user qutoes #7854`.

File: src/main/webapp/index.jsp [hunk 1]
```diff
@@ -37,8 +37,8 @@
     <div class="col-xs-12 col-sm-8">
       <h2 class="media-heading">Praised by Users:</h2>
       <p id="testimonialContainer">
-        Thank you for building such a wonderful tool.
-        <br>-Faculty user, Canada
+        I just started exploring TEAMMATES and am very impressed. Wish I discovered it earlier.
+        <br>Faculty user, Singapore
```
Comment (chowyb) [inline src/main/webapp/index.jsp]: Use `<br>-Faculty user, Singapore` or `<br>- Faculty user, Singapore`. This applies to the strings in the .es6 file as well.

If using the `<br>- Faculty user, Singapore` variation (with a space between the dash and the next word), then update all strings in the .es6 file to match this convention.

**Comment by kyleaolson** [comment]
Thanks for the suggestion, updated!

File: src/main/webapp/index.jsp [hunk 2]
```diff
@@ -37,8 +37,8 @@
     <div class="col-xs-12 col-sm-8">
       <h2 class="media-heading">Praised by Users:</h2>
       <p id="testimonialContainer">
-        Thank you for building such a wonderful tool.
-        <br>-Faculty user, Canada
+        I just started exploring TEAMMATES and am very impressed. Wish I discovered it earlier.
+        <br>-Faculty user, Singapore
```
Comment (wkurniawan07) [inline src/main/webapp/index.jsp]: This is a repetition from an existing line in `index.es6`, and it goes against DRY principle.

Can you remove these two lines and add `loopTestimonials()` in `index.es6`, just before the line that contains `window.setInterval(loopTestimonials, LOOP_INTERVAL);`?

**Pull request merged**